### PR TITLE
rename every instance of "stable" to "legacy"

### PR DESCRIPTION
**Pull request by cobalt2727**
to the best of my knowledge this is only done on instances of the entire word, matching case sensitivity

I'm not sure if/how this would break non-English languages, feedback on that is welcome

### CODE REVIEW

**Comment by MayImilae** [comment]
The intention is to change stables to releases in the future, but that is indefinitely stalled out so this is a good option to clear up confusion in the meantime.

BTW you probably should also change the stable track in the updater to legacy too. Will you be doing that in a separate PR?

**Comment by OatmealDome** [comment]
Does anyone know how is localization handled on the website? I know for the main emulator it is handled by Transifex and you're not supposed to change the strings in the `po` files manually. Not sure if it's the same here.

**Comment by cobalt2727** [comment]
> The intention is to change stables to releases in the future, but that is indefinitely stalled out so this is a good option to clear up confusion in the meantime.
> 
> BTW you probably should also change the stable track in the updater to legacy too. Will you be doing that in a separate PR?

I can look into that when I get some time, although I'm thinking that might take more work than a "replace all" in VS Code...

**Comment by JosJuice** [comment]
> Does anyone know how is localization handled on the website? I know for the main emulator it is handled by Transifex and you're not supposed to change the strings in the `po` files manually. Not sure if it's the same here.

I believe it's the same here. However, the process of syncing the strings with Transifex has been handled by delroth instead of by me.

**Comment by Dentomologist** [comment]
FYI, Stable was already removed from the updater in https://github.com/dolphin-emu/dolphin/pull/11457.

**Comment by MayImilae** [comment]
> FYI, Stable was already removed from the updater in [dolphin-emu/dolphin#11457](https://github.com/dolphin-emu/dolphin/pull/11457).

Oh, I forgot about that. Nevermind then!

**Comment by OatmealDome** [comment]
@JosJuice Are you able to handle a Transifex sync for this repo too? I assume this will be the same process, though I have no experience with Transifex.

File: fabfile.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def deploy(c, root, branch):
 
 @task(hosts=_HOSTS)
 def deploy_stable(c):
-    deploy(c, "/home/dolphin-emu/apps/www", "stable")
+    deploy(c, "/home/dolphin-emu/apps/www", "legacy")
```
Comment (OatmealDome) [inline fabfile.py]: Please do not change this instance of "stable". It is referring to the `git` branch name on this repository, not Dolphin stable.

**Review by OatmealDome — CHANGES_REQUESTED** [review]

**Comment by JosJuice** [comment]
> Are you able to handle a Transifex sync for this repo too? I assume this will be the same process, though I have no experience with Transifex.

The exact process would be a little different since this is a different code base in a different programming language, , but I will try to look into it when I get time. It's probably not too complicated.

File: fabfile.py (hunk 1 above)
Comment (cobalt2727) [reply fabfile.py]: any other instances like this?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Use of Git
 ----------
 
 The Git branch deployed on the main website instance (https://dolphin-emu.org/)
-is the ``stable`` branch. When we do large scale changes that require staging
+is the ``legacy`` branch. When we do large scale changes that require staging
```
Comment (OatmealDome) [inline CONTRIBUTING.md:19]: This is another instance that shouldn't be changed.

**Review by OatmealDome — CHANGES_REQUESTED** [review]
Please drop all changes to the `po` files. Transifex will handle this.

**Comment by OatmealDome** [comment]
Superseded by #152.

**Pull request closed without merging**